public/app/features/dashboard/components/PanelEditor/PanelOptionsTab.tsx

```diff
@@ -46,7 +73,50 @@ export const PanelOptionsTab: FC<Props> = ({
       visTabInputRef.current.focus();
     }
   };
-  // Fist common panel settings Title, description
+
+  const useLibraryPanel = ({ model, uid, title, lastEdited, lastAuthor }: LibraryPanelCardProps) => {
```

Comment:
Would be nice to move the library panel stuff out to it's own component thís file & component is already quite big